src/main/java/com/aws/greengrass/clientdevices/auth/configuration/SecurityConfiguration.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.configuration;
+
+import com.aws.greengrass.config.Topics;
+import com.aws.greengrass.logging.api.Logger;
+import com.aws.greengrass.logging.impl.LogManager;
+import com.aws.greengrass.util.Coerce;
+import lombok.Getter;
+
+import java.util.Objects;
+
+/**
+ * Represents the security configuration. Acts as an adapter from the GG Topics to the domain.
+ * <p>
+ * |---- configuration
+ * |    |---- security:
+ * |          |---- clientDeviceTrustDurationMinutes: "..."
+ * </p>
+ */
+@Getter
+public final class SecurityConfiguration {
+    private static final Logger logger = LogManager.getLogger(SecurityConfiguration.class);
+    public static final String SECURITY_TOPIC = "security";
+    public static final String CLIENT_DEVICE_TRUST_DURATION_MINUTES_TOPIC = "clientDeviceTrustDurationMinutes";
+    public static final int DEFAULT_CLIENT_DEVICE_TRUST_DURATION_MINUTES = 24;
```

Comment:
24?

Reply:
Yeah I thought it would be 60 minutes

Reply:
We wanted this to be opt in, so 0 by default

Reply:
That is correct. @vaibhavmurkute mentioned he would like to handle it on a separate PR given setting it to 0 basically rejects everything, event when requesting things from the cloud at the moment... There are some minor tweaks that need to happen to make this a reality

Reply:
yeah, will update this to an _opt-in_ kinda default (0) in a separate PR.